### PR TITLE
Add autoconfiguration of bigquery connection in CloudBeaver.

### PR DESCRIPTION
**Pull request by JDarDagran**
Signed-off-by: Jakub Dardzinski <kuba0221@gmail.com>

### CODE REVIEW

File: jupyterlab-dataops/resources/cloudbeaver/conf/initial-data-sources-bigquery.conf.j2 [hunk 1]
```diff
@@ -11,10 +11,11 @@
 			"configuration": {
 				"host": "https://www.googleapis.com/bigquery/v2",
 				"port": "443",
+				"database": "{{ project_id }}",
 				"url": "jdbc:bigquery://https://www.googleapis.com/bigquery/v2:443",
 				"type": "dev",
 				"properties": {
-					"OAuthPvtKeyPath": "/home/jupyter/gcp-service-account-key.json"
+					"OAuthType": "3"
```
Comment (p-pekala) [inline jupyterlab-dataops/resources/cloudbeaver/conf/initial-data-sources-bigquery.conf.j2:18]: What is this "3"?

File: jupyterlab-dataops/resources/cloudbeaver/set_metadata.py [hunk 2]
```diff
@@ -1,6 +1,10 @@
 from google.cloud import compute_v1
```
Comment (p-pekala) [inline jupyterlab-dataops/resources/cloudbeaver/set_metadata.py:1]: when this script is executed?

**Review by p-pekala — APPROVED** [review]

File: jupyterlab-dataops/resources/cloudbeaver/set_metadata.py (hunk 2 above)
Comment (JDarDagran) [reply jupyterlab-dataops/resources/cloudbeaver/set_metadata.py:1]: it's called from Bash script at startup of the image. Please check CMD in Dockerfile.

File: jupyterlab-dataops/resources/cloudbeaver/conf/initial-data-sources-bigquery.conf.j2 (hunk 1 above)
Comment (JDarDagran) [reply jupyterlab-dataops/resources/cloudbeaver/conf/initial-data-sources-bigquery.conf.j2:18]: That's enum for authenticating with Google Authentication Default Credentials. I can provide link with docs for JDBC BigQuery driver in comment above.

**Pull request merged**